src/components/common/Extensions/ColonySwitcher/ColoniesDropdown.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC } from 'react';
```

Comment:
Please, create a partials directory where you put smaller parts of this component